.sync/devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "image": "ghcr.io/tianocore/containers/fedora-37-dev:latest",
-  "postCreateCommand": "git config --global --add safe.directory * && pip install --upgrade -r pip-requirements.txt",
+  "postCreateCommand": "git config --global --add safe.directory '*' && pip install --upgrade -r pip-requirements.txt",

```

Comment:
According to [this SO post](https://stackoverflow.com/a/71904131), double quotes might be needed on Windows. Did you see an issue like that?

Reply:
We can go with that for now.